libs/interpret-text/src/lib/TextExplanationDashboard/Control/BarChart/getTokenImportancesChartOptions.ts

```diff
@@ -46,6 +61,26 @@ export function getTokenImportancesChartOptions(
     ylabel.push(props.text[idx]);
     tooltip.push(str);
   });
+
+  // add output feature importance
+  if (props.outputFeatureValue && props.baseValue) {
+    const outputFeatureValueIndex = findNearestIndex(
+      x,
+      props.outputFeatureValue
+    );
+    const baseValueFeatureValueIndex = findNearestIndex(x, props.baseValue);
+    if (outputFeatureValueIndex && baseValueFeatureValueIndex) {
+      if (Utils.addItem(props.outputFeatureValue, props.radio)) {
+        x.splice(outputFeatureValueIndex, 0, props.outputFeatureValue);
+        ylabel.splice(outputFeatureValueIndex, 0, outputFeatureImportanceLabel);
+      }
+      if (Utils.addItem(props.baseValue, props.radio)) {
+        x.splice(baseValueFeatureValueIndex, 0, props.baseValue);
+        ylabel.splice(baseValueFeatureValueIndex, 0, baseValueLabel);
+      }
```

Comment:
these two ifs look very similar - I would refactor into separate helper method and use different params for props.baseValue/props.outputFeatureValue and baseValueFeatureValueIndex/outputFeatureValueIndex